rules/windows/process_creation/proc_creation_win_rar_susp_greedy.yml

```diff
@@ -0,0 +1,40 @@
+title: RAR Greedy Compression 
+id: afe52666-401e-4a02-b4ff-5d128990b8cb
```

Comment:
This rule was already pushed in private with the same ID. I guess we move it to the public repo now.

Reply:
Yes

Reply:
Ok. Deleted